src/main/resources/static/js/pollingunits.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const React = require('react');
+const ReactDOM = require('react-dom');
+const client = require('./client');
+
+class PollingUnitsComponent extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {pollingunits: []};
+	}
+	componentDidMount() {
+		client({method: 'GET', path: '/api/pollingUnits'}).done(response => {
+			this.setState({pollingunits: response.entity._embedded.pollingUnits});
+		});
+	}
+	render() {
+		return (
+			<PollingUnits pollingunits={this.state.pollingunits}/>
+		)
+	}
+}
+class PollingUnits extends React.Component{
+	render() {
+		//if (this.props.data) {
+			const pollingunits = this.props.pollingunits.map(pollingunit =>
+				<PollingUnit key={pollingunit._links.self.href} pollingunit={pollingunit}/>
+			);
+		//}
+		return (
+			<table>
```

Comment:
You might want to take a look at your text editor, the indentation here is not properly formed

Reply:
@sfagade this is actually a long line that spilled to another line.  Also, the commented lines have an effect.  Should be OK in the next PR.  Thanks.